Gruntfile.js

```diff
@@ -16,13 +16,23 @@ var _ = require("lodash");
 var path = require("path");
 var execSync = require("child_process").execSync;
 
+var getFromExec = function (command) {
+    var result;
+    try {
+        result = execSync(command);
+    } catch (e) {
+        result = "unknown\n";
```

Comment:
I wonder what is captured in the argument ```e``` when an error occurs. Does it make sense to return an error message or an error code or both if there's any returned by ```e```.